manifests/plugin/remote_execution/cockpit.pp

```diff
@@ -0,0 +1,53 @@
+# Installs remote execution cockpit plugin
+class foreman::plugin::remote_execution::cockpit {
+  require foreman::plugin::remote_execution
+
+  $config_directory = '/etc/foreman/cockpit'
+  $foreman_url = $foreman::foreman_url
+  $cockpit_path = '/webcon'
+  $cockpit_host = '127.0.0.1'
+  $cockpit_port = 9999
+  $cockpit_config = {
+    'foreman_url' => $foreman_url,
+    'ssl_ca_file' => $foreman::server_ssl_chain,
+    'ssl_certificate' => $foreman::client_ssl_cert,
+    'ssl_private_key' => $foreman::client_ssl_key,
+  }
+
+  foreman::plugin { 'remote_execution-cockpit': }
+  -> service { 'foreman-cockpit':
+    ensure => running,
+    enable => true,
+  }
+
+  file { "${config_directory}/cockpit.conf":
+    ensure  => file,
+    owner   => 'root',
+    group   => 'root',
+    mode    => '0644',
+    content => template('foreman/remote_execution_cockpit.conf.erb'),
+    require => Foreman::Plugin['remote_execution-cockpit'],
+    notify  => Service['foreman-cockpit'],
+  }
+
+  file { "${config_directory}/foreman-cockpit-session.yml":
+    ensure  => file,
+    owner   => 'root',
+    group   => 'root',
+    mode    => '0644',
+    content => template('foreman/remote_execution_cockpit_session.yml.erb'),
+    require => Foreman::Plugin['remote_execution-cockpit'],
+    notify  => Service['foreman-cockpit'],
```

Comment:
I am wondering a bit if this service name should be more specific. As I think I understand the comments, this is dependent entirely upon remote execution and is not a generic service for Foreman and cockpit. Something like `foreman-rex-cockpit` or `foreman-remote-execution-cockpit`.

Reply:
In that case we should update it in packaging first since this mirrors that:
https://github.com/theforeman/foreman-packaging/blob/e61fc0b8bc36d9658bfa7bb4177aca60834a53e6/packages/plugins/rubygem-foreman_remote_execution/rubygem-foreman_remote_execution.spec#L95